internal/controllers/user_devices_controller.go

```diff
@@ -1668,7 +1917,8 @@ func (udc *UserDevicesController) UpdateNFTImage(c *fiber.Ctx) error {
 type VehicleMintRequest struct {
 	NFTImageData
 	// Signature is the hex encoding of the EIP-712 signature result.
-	Signature string `json:"signature" validate:"required"`
+	Signature string               `json:"signature" validate:"required"`
+	SACDInput *contracts.SacdInput `json:"sacd_input,omitempty"`
```

Comment:
Let's add a little documentation here. We will certainly have to regenerate Swagger.